lib/internal-rules/internal-no-invalid-meta.js

```diff
@@ -0,0 +1,208 @@
+/**
+ * @fileoverview Internal rule to prevent missing or invalid meta property in core rules.
+ * @author Vitor Balocco
+ */
+
```

Comment:
Don't forget the section separators:http://eslint.org/docs/developer-guide/code-conventions#file-format